### PR TITLE
docs: indentation fixes (master)

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: src/borg/archiver.py [hunk 1]
```diff
@@ -2237,7 +2237,7 @@ def do_break_lock(self, args, repository):
         File patterns support these styles: fnmatch, shell, regular expressions,
         path prefixes and path full-matches. By default, fnmatch is used for
         ``--exclude`` patterns and shell-style is used for the ``--pattern``
-         option. For commands that support patterns in their ``PATH`` argument
```
Comment (hexagonrecursion) [inline src/borg/archiver.py:2240]: Regression introduced in 5d4dda75d36d4bdaff9fc4a38a234383aef4b733 - nothing to backport

File: src/borg/archiver.py [hunk 2]
```diff
@@ -2358,64 +2358,64 @@ def do_break_lock(self, args, repository):
             EOF
             $ borg create --exclude-from exclude.txt backup /
```
Comment (hexagonrecursion) [inline src/borg/archiver.py:2360]: The below regression was introduced in 5d4dda75d36d4bdaff9fc4a38a234383aef4b733 - nothing to backport

File: src/borg/archiver.py (hunk 1 above)
Comment (ThomasWaldmann) [reply src/borg/archiver.py:2240]: yup.

File: src/borg/archiver.py (hunk 2 above)
Comment (ThomasWaldmann) [reply src/borg/archiver.py:2360]: do you mean the bigger block with bad indentation? i don't see it there.

Comment (hexagonrecursion) [reply src/borg/archiver.py:2360]: Sorry. My comment got stuck because of the github review feature and I did not notice.

Comment (ThomasWaldmann) [reply src/borg/archiver.py:2360]: Thanks for the hint!

Comment (ThomasWaldmann) [reply src/borg/archiver.py:2360]: removed backport label, nothing to do here...

**Pull request merged**